packages/desktop-client/src/components/reports/reports/SpendingCard.tsx

```diff
@@ -46,8 +54,36 @@ export function SpendingCard() {
       data.intervalData[todayDay][spendingReportCompare];
   const showLastMonth = data && Math.abs(data.intervalData[27].lastMonth) > 0;
 
+  const spendingReportFeatureFlag = useFeatureFlag('spendingReport');
+
+  if (!spendingReportFeatureFlag) {
+    return (
+      <MissingReportCard isEditing={isEditing} onRemove={onRemove}>
+        The experimental spending report feature has not been enabled.
```

Comment:
Should this be translatable?  I noticed you had t() calls in some places, but it seemed inconsistent.

I guess moving forward most user-faceable strings should be wrapped for translation?

Reply:
Yea, I was just checking out how tricky adding translations would be in a single file. Definitely didn't add them everywhere :)